editions/tw5tiddlyweb/tiddlywiki.info

```diff
@@ -1,7 +1,8 @@
 {
```

Comment:
The TiddlyWeb spec is much more then the basic tiddlyweb adapter implemented in TW5. ... So imo it should have a different name. 

I think "webdav-adaptor" would be OK for me. ... So the PUT_save we implemented could be: "webdav-put-etag" and a version I'm experimenting with could be "webdav-put-modified" ... Which uses "if-unmodified-since" header. IMO it's much more robust against different server settings as "compression" on/off and so on.